jupyterlab-dataops/Dockerfile

```diff
@@ -97,12 +97,14 @@ RUN pip install --no-cache-dir \
     jupyterlab-git \
     jupyterlab-lsp \
     jupyter-server-proxy \
-    data-pipelines-cli[docker,datahub,s3,snowflake]==${DP_CLI_VERSION}
+    data-pipelines-cli[docker,datahub,s3,snowflake]==${DP_CLI_VERSION} \
+    sqlfluff==1.4.5
 
 # install code server and addons
 RUN curl -fsSL https://code-server.dev/install.sh | sh -s -- --version ${VSCODE_VERSION}
 RUN code-server --extensions-dir /var/tmp/extension --install-extension ms-python.python \
     code-server --extensions-dir /var/tmp/extension --install-extension innoverio.vscode-dbt-power-user \
-    code-server --extensions-dir /var/tmp/extension --install-extension /var/tmp/snowflake.snowflake-vsc-0.5.1.vsix
+    code-server --extensions-dir /var/tmp/extension --install-extension /var/tmp/snowflake.snowflake-vsc-0.5.2.vsix \
```

Comment:
Version mismatch or it was on purpose?

Reply:
The newest version which is `0.5.2` gives us the possibility to use SSO